### PR TITLE
Using `Carbon\CarbonInterface` instead of `Illuminate\Support\Carbon`

### PR DESCRIPTION
**Pull request by leandrogehlen**
This allows using `Carbon\CarbonImmutable`

### CODE REVIEW

**Comment by sawirricardo** [comment]
Any ETA when this will be merged?

File: composer.json [hunk 1]
```diff
@@ -44,7 +44,10 @@
         "test-coverage": "vendor/bin/phpunit --coverage-html coverage"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "pestphp/pest-plugin": true
```
Comment (buddhaCode) [inline composer.json:49]: Think this has nothing to do with the subject of this PR.

Comment (leandrogehlen) [reply composer.json]: I have already reverted the compose

**Review by buddhaCode — APPROVED** [review]

**Comment by sawirricardo** [comment]
when will this be merged?

**Comment by buddhaCode** [comment]
ping @Larsklopstra ;)

**Comment by Larsklopstra** [comment]
Great addition, thanks!

**Pull request merged**